sql-plugin/src/main/301until310-all/scala/com/nvidia/spark/rapids/shims/v2/OffsetWindowFunctionMeta.scala

```diff
@@ -14,62 +14,47 @@
  * limitations under the License.
  */
 
-package com.nvidia.spark.rapids.shims.spark313
+package com.nvidia.spark.rapids.shims.v2
```

Comment:
this whole file is a very odd diff, it did not get moved from 313 ithis got copied from GpuWindowExpression.scala